### PR TITLE
add lesson plan print all flow

### PR DESCRIPTION
**Pull request by ZarishIqbal**
download all lesson plans

### CODE REVIEW

File: web/src/modules/TIP/LessonPlans/List.tsx [hunk 1]
```diff
@@ -160,6 +160,11 @@ const List = ({ match, history }: RouteComponentProps) => {
 
 	const lesson_progress = getLessonProgress(faculty[facultyId])
 
+	const printAll = () => {
+		Object.entries(lessonPlanBooklets[class_name] ?? {}).forEach(([subject, link]) => {
+			downloadLink(`${class_name}_${subject}.pdf`, link, printRef)
+		})
```
Comment (mudssrali) [inline web/src/modules/TIP/LessonPlans/List.tsx:166]: There are twelve combinations we have for each group. Lesson pdf should be printed for visited class/subject, not for all class' subjects, no?

File: web/src/modules/TIP/TIPConstants.ts [hunk 2]
```diff
@@ -5,3 +5,40 @@ export enum Types {
 	SINGLE_QUIZ_VIEW,
 	SINGLE_SLO_VIEW
 }
+type booklets = {
+	[level in TIPLevels]: { [subject in TIPSubjects]: string }
+}
+export const lessonPlanBooklets: Partial<booklets> = {
+	'Level KG': {
+		English:
+			'https://storage.googleapis.com/targeted-instructions/Lesson%20Plans/booklets/Eng%20Level%20KG%20Lessons_210816.pdf',
```
Comment (mudssrali) [inline web/src/modules/TIP/TIPConstants.ts:14]: we should avoid having links with `%20` (space)

**Review by mudssrali — CHANGES_REQUESTED** [review]
LGTM - Thanks @ZarishIqbal. I left a question and a suggestion.

File: web/src/modules/TIP/LessonPlans/List.tsx (hunk 1 above)
Comment (ZarishIqbal) [reply web/src/modules/TIP/LessonPlans/List.tsx]: You're right. 
I fixed it.

File: web/src/modules/TIP/TIPConstants.ts (hunk 2 above)
Comment (ZarishIqbal) [reply web/src/modules/TIP/TIPConstants.ts]: Done.

**Review by mudssrali — APPROVED** [review]
Thanks @ZarishIqbal

**Pull request merged**